app/views/network_router/remove_interface_select.html.haml

```diff
@@ -0,0 +1,27 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "networkRouterFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Remove Interface from Router')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet')
+      .col-md-8
+        = select_tag("cloud_subnet_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @subnet_choices.sort),
+                      "ng-model"                    => "networkRouterModel.cloud_subnet_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
+          = render :partial => "layouts/angular/x_custom_form_buttons_angular",
+                   :locals  => {:button_label           => _("Remove"),
+                                :button_click           => "removeInterfaceClicked()"}
```

Comment:
@sseago I know this part with `%table ...` was copied from other places in our views,
but I fixed that recently -- we don't want to use table for styling here, so please fix
this similarly to what I did here:

https://github.com/ManageIQ/manageiq-ui-classic/pull/78/files#diff-64b58413d95ff03ddc058038d06cae9dR17